src/extension.ts

```diff
@@ -352,13 +359,19 @@ export async function activate(context: extensionApi.ExtensionContext): Promise<
         },
       )
       .then(() => false)
-      .catch(() => true); // required to force Promise.all() call keep waiting for all not failed calls
+      .catch((error) => {
+        telemetryData['vmSubscriptionActivationError'] = String(error);
+        return true; // required to force Promise.all() call keep waiting for all not failed calls
+      });
 
     const failed = await Promise.all([registryAccess, vmActivation]); // wait for all fail or pass
 
     if (failed.some(fail => fail)) {
       removeSession(currentSession.id); // if at least one fail, remove session
     }
+
+    TelemetryLogger.logUsage('redhat.authentication.signin', telemetryData);
```

Comment:
For consistency and accuracy, I think the name of the event should be using the name of the extension:
`redhat.redhat-authentication`

Reply:
The extension id is added by the Podman Desktop API so not required here. signin should be enough

Reply:
Updated logUsage call to use short event names like `signin`, `signout` and etc